### PR TITLE
Removed media queries and uesd bootstrap class in about section

### PR DESCRIPTION
**Pull request by abhijeetchauhan**

### CODE REVIEW

File: src/client/css/style.css [hunk 1]
```diff
@@ -183,52 +183,66 @@ section#About{
...
+section#community .col-md-4{
+  padding-bottom: 40px;
 }
-@media(max-width: 622px){
-  /*---About Section----*/
-  #coreContributers .bigImages{
-  display: none;
-  }
-  /*---End---*/
+section#community .contribute-btn{
```
Comment (handasolo) [inline src/client/css/style.css:210]: Please add `color:white` and `font-weight:bold`

File: src/client/index.html [hunk 2]
```diff
@@ -51,25 +53,25 @@
...
           <div class="row">
-            <div id="AboutUs" class="col-md-12">
+            <div id="AboutUs" class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
               <h1>About us</h1>
               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eget dui eget elit semper aliquet. Fusce nec mi sapien. Nulla consectetur lorem in libero pellentesque, eget consectetur mi lobortis. Integer efficitur, dui at molestie consectetur, nibh nisl feugiat ex, eu condimentum nisi lorem at felis. Integer in mollis diam. Praesent eget malesuada lacus, id rutrum lacus. Donec eget euismod libero. </p>
             </div>
-            <div id="coreContributers" class="col-md-12">
-            <span class="bigImages">
+            <div id="coreContributers" class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <span class="bigImages hidden-xs">
```
Comment (handasolo) [inline src/client/index.html]: please add `class="img-circle"` to all `img`

File: src/client/index.html [hunk 3]
```diff
@@ -51,25 +53,25 @@
...
               <img src="http://placehold.it/150x150" alt="contributer1">
               <img src="http://placehold.it/150x150" alt="contributer2">
               <img src="http://placehold.it/150x150" alt="contributer3">
               <img src="http://placehold.it/150x150" alt="contributer4">
               <img src="http://placehold.it/150x150" alt="contributer5">
-              </span>
-              <span class="smallImages">
+            </span>
+            <span class="smallImages visible-xs">
```
Comment (handasolo) [inline src/client/index.html]: Same with this

File: src/client/index.html [hunk 4]
```diff
@@ -51,25 +53,25 @@
...
-              </span>
+            <div id="coreContributers" class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <span class="bigImages hidden-xs">
+              <img src="http://placehold.it/150x150" alt="contributer1" class="img-circle">
+              <img src="http://placehold.it/150x150" alt="contributer2" class="img-circle">
+              <img src="http://placehold.it/150x150" alt="contributer3" class="img-circle">
+              <img src="http://placehold.it/150x150" alt="contributer4" class="img-circle">
+              <img src="http://placehold.it/150x150" alt="contributer5" class="img-circle">
+            </span>
+            <span class="smallImages visible-xs img-circle">
```
Comment (handasolo) [inline src/client/index.html]: Remove all the small images.

Comment (handasolo) [inline src/client/index.html]: They look just fine at 150x150

**Comment by Slayslot** [comment]
Merging

**Comment by handasolo** [comment]
merged

**Pull request merged**